kratos/python_scripts/restart_utility.py

```diff
@@ -26,7 +29,8 @@ def __init__(self, model_part, settings):
             "restart_save_frequency"         : 0.0,
             "restart_control_type"           : "time",
             "save_restart_files_in_folder"   : true,
-            "set_mpi_communicator"           : true
+            "set_mpi_communicator"           : true,
+            "max_files_to_keep"              : -1
```

Comment:
Mate I made it consistent with the hdf app, can you please update the rest of the code?

Reply:
the naming and behavior should be consistent with #7618

Reply:
I made some minor updates. I like the "-1" slightly better than what is used in HDF5, so I would keep it. I do not see major changes. 

@sunethwarna ?

Reply:
I think I was going with string `"unlimited"` because in HDF5 they automatically do the conversion from `Kratos.Parameters` to `int` or `string` depending on the type given in json (no need to use `GetInt` or `GetString`. In here it makes sense to use -1 to keep the code simple. I will change the HDF5 implementation also to follow the same syntax to be consistent. :)